test/test-info.cpp

```diff
@@ -2,14 +2,18 @@
 
 #include "mapget/model/info.h"
 #include "mapget/model/stream.h"
-
-#include <iostream>
+#include "mapget/log.h"
 
 using namespace mapget;
 
 TEST_CASE("InfoToJson", "[DataSourceInfo]")
 {
-    // Create a DataSourceInfo object
+    // Log all messages to the console if MAPGET_LOG_FILE is not specified.
+    if (getenv("MAPGET_LOG_FILE") == nullptr) {
+        setenv("MAPGET_LOG_LEVEL", "trace", 1);
```

Comment:
Maybe we can directly use `log()->set_level()` here?

Reply:
I will set the level to trace, since it's just test cases.